src/js/components/Select/SelectContainer.js

```diff
@@ -279,6 +279,18 @@ const SelectContainer = forwardRef(
             } else {
               label = e;
             }
+
+            if (optionsRef.current) {
+              if (
+                optionsRef.current.childNodes[
+                  index
+                ].childNodes[0].childNodes[0].textContent
+                  .charAt(0)
+                  .toLowerCase() === event.key.toLowerCase()
+              ) {
+                optionsRef.current.childNodes[index].scrollIntoView(true);
+              }
+            }
```

Comment:
This works with the Select/Simple story but not the Select/ManyOptions story. I swapped the array in both stories to be a list of letters in the alphabet so that it was easier to test. In the ManyOptions story I got the following error: `Uncaught TypeError: Cannot read property 'childNodes' of undefined`. I think the code is too specific to one use case of select.